test_runner/sql_regress/sql/neon-subxacts.sql

```diff
@@ -0,0 +1,16 @@
+DO $$
+DECLARE
+i numeric;
+BEGIN
+  FOR i IN 1..1_000_000 LOOP
+    BEGIN
+      PERFORM 1;
+    EXCEPTION WHEN OTHERS THEN
+      RAISE WARNING 'error';
+    END;
```

Comment:
Could this be tested with some DDL? E.g. update a nologin user's password to '' every 100k iterations or so?

Reply:
Thanks!